web/packages/teleport/src/ironrdp/src/lib.rs

```diff
@@ -211,12 +212,17 @@ impl FastPathProcessor {
                     UpdateKind::Region(region) => {
                         outputs.push(ActiveStageOutput::GraphicsUpdate(region));
                     }
-                    UpdateKind::PointerDefault
-                    | UpdateKind::PointerHidden
-                    | UpdateKind::PointerPosition { .. }
-                    | UpdateKind::PointerBitmap(_) => {
-                        warn!("Pointer updates are not supported");
-                        continue;
+                    UpdateKind::PointerDefault => {
+                        outputs.push(ActiveStageOutput::PointerDefault);
+                    }
+                    UpdateKind::PointerHidden => {
+                        outputs.push(ActiveStageOutput::PointerHidden);
+                    }
+                    UpdateKind::PointerPosition { x, y } => {
+                        outputs.push(ActiveStageOutput::PointerPosition { x: x, y: y });
+                    }
```

Comment:
If we're handling PointerHidden, then we should probably handle PointerDefault as well, since that seems to mean "unhide": https://github.com/Devolutions/IronRDP/blob/c944674ecbb0952f9e7ecb964a6c79cdca669a08/crates/ironrdp-client/src/gui.rs#L247-L249

And we may as well handle PointerPosition too, or if that's a lot of extra work then at least add some warning logging for ourselves.

Reply:
We can't handle PointerPosition as we can't move mouse pointer from browser. As for PointerDefault we'd have to remember what we had before to restore it after "unhide", doable, I'll try to handle it when I'm back

Reply:
> We can't handle PointerPosition as we can't move mouse pointer from browser.

Sounds good, in that case let's just add some sort of warning logging for exhaustiveness / debugging in the odd chance it becomes problematics in the future.

> As for PointerDefault we'd have to remember what we had before to restore it after "unhide", doable, I'll try to handle it when I'm back

Sounds good.

Reply:
Added a warning for position updates and handled `PointerDefault` message